receiver/README.md

```diff
@@ -202,11 +254,63 @@ with detailed sample configurations [here](zipkinreceiver/testdata/config.yaml).
 
 # <a name="metric-receivers"></a>Metric Receivers
 
+## <a name="hostmetrics"></a>Host Metrics Receiver
+
+The Host Metrics receiver generates metrics about the host system. This is
+intended to be used when the collector is deployed as an agent.
+
+The categories of metrics scraped can be configured under the `scrapers` key.
+For example:
+
+```yaml
+hostmetrics:
+  collection_interval: 1m
+  scrapers:
+    cpu:
+    memory:
+    disk:
+```
+
+If you would like to scrape some metrics at a different frequency than others,
+you can configure multiple `hostmetrics` receivers with different
+`collection_interval values`. For example:
+
+```yaml
+receivers:
+  hostmetrics:
+    collection_interval: 30s
+    scrapers:
+      cpu:
+      memory:
+
+  hostmetrics/disk:
+    collection_interval: 1m
+    scrapers:
+      disk:
+      filesystem:
+
+service:
+  pipelines:
+    metrics:
+      receivers: [hostmetrics, hostmetrics/disk]
+```
+
+### OS Support
+
+The initial implementation of the Host Metrics receiver supports Windows only.
```

Comment:
This is wrong because for everything we implemented so far we support windows and Unix systems.

Reply:
Pls remove this OS Support section completely for now. Note this receiver is still WIP but I intend to come back and update / add to this documentation once its a little further along.

Thanks for moving these docs to be consistent btw.